docs/metrics.md

```diff
@@ -38,7 +38,17 @@ The following metrics are added to the ping:
 
 | Name | Type | Description | Data reviews | Extras | Expiration |
 | --- | --- | --- | --- | --- | --- |
+| controls.context_menu_long_pressed |[counter](https://mozilla.github.io/glean/book/user/metrics/counter.html) |Counting how many times users do a long press on the context menu.  |[1](https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568)||2020-05-01 |
```

Comment:
This is out of date. Did you commit after you added `pings.yaml`? It should have documented the changes here.